plugins/woocommerce/includes/cli/class-wc-cli-com-extension-command.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * WC_CLI_COM_Command class file.
+ *
+ * @package WooCommerce\CLI
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'Plugin_Command' ) ) {
+	exit;
+}
+
+/**
+ * Allows to interact with extensions from WCCOM marketplace via CLI.
+ *
+ * @version 6.8
+ * @package WooCommerce
+ */
+class WC_CLI_COM_Extension_Command extends Plugin_Command {
+	/**
+	 * Registers a commands for managing WooCommerce.com extensions.
+	 */
+	public static function register_commands() {
+		WP_CLI::add_command( 'wc com extension', 'WC_CLI_COM_Extension_Command' );
+	}
+
+	/**
+	 * List extensions available for the connected site.
+	 *
+	 * ## EXAMPLES
+	 *
+	 * wp wc com extension list
+	 *
+	 * @param array $args WP-CLI positional arguments.
+	 * @param array $assoc_args WP-CLI associative arguments.
+	 */
+	public function list_( $args, $assoc_args ) {
```

Comment:
I think we can remove the list command from here and add changes from https://github.com/woocommerce/woocommerce/pull/33783 to this file

Reply:
removed in bcdf636